{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/.github/workflows/run-tests-fs.yml

```diff
@@ -120,11 +120,11 @@ jobs:
                   }
                 ],
                 "notebook_task": {
-                  "notebook_path": "{{cookiecutter.project_name}}/feature_engineering/notebooks/TrainWithFeatureStore",
+                  "notebook_path": "{{cookiecutter.project_name}}/training/notebooks/TrainWithFeatureStore",
                   "base_parameters": {
-                    "test_mode": "True",
+                    "env": "staging"
                     "training_data_path": "/databricks-datasets/nyctaxi-with-zipcodes/subsampled",
-                    "experiment_name": "{{cookiecutter.mlflow_experiment_parent_dir}}/{{cookiecutter.experiment_base_name}}-test",
+                    "experiment_name": "{{cookiecutter.mlflow_experiment_parent_dir}}-staging/{{cookiecutter.experiment_base_name}}-test",
```

Comment:
Aren't we supposed to use dev workspace for testing(CI)?

Reply:
The `DATABRICKS_HOST` value specified in the file is `staging` so I just used that. We don't technically take in a dev workspace parameter in stacks so setting up a dev workflow from default would involve that

Reply:
If in test model(not initiated from terraform workflow), we may want to just use "/{{cookiecutter.experiment_base_name}}-test".
Because if user run the notebook from dev workspace, the folder  `{{cookiecutter.mlflow_experiment_parent_dir}}` or `{{cookiecutter.mlflow_experiment_parent_dir}}-staging` will not exist. The experiment name `{cookiecutter.mlflow_experiment_parent_dir}}-staging/<name>` may cause error.